doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -2,243 +2,33 @@
-- Create a branch to work in.
-
-## Create starter package  
-
-For this guide, we'll create a getting started project in a branch of your fork of `azure-sdk-for-net` repo. The started project will be under `sdk\<servie name>\<package name>` directory of `azure-sdk-for-net` repo. The package will contain several folders and files (see following). Please refer to [sdk-directory-layout](https://github.com/Azure/azure-sdk/blob/main/docs/policies/repostructure.md#sdk-directory-layout) for detail information.
-
-```text
-sdk\<service name>\<package name>\README.md
-sdk\<service name>\<package name>\api
-sdk\<service name>\<package name>\perf
-sdk\<service name>\<package name>\samples
-sdk\<service name>\<package name>\src
-sdk\<service name>\<package name>\stress
-sdk\<service name>\<package name>\tests
-sdk\<service name>\<package name>\CHANGELOG.md
-sdk\<service name>\<package name>\<package name>.sln
-```
-
-- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate
-- `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
-
-### Use Cadl as Input
-
-#### Create Cadl project
-  
-  You can download existing cadl project from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow following steps to create a new cadl project.
-
-  ***Initialize Cadl Project***
-
-  Follow [Cadl Getting Start](https://github.com/microsoft/cadl/#using-node--npm) to initialize your Cadl project.
-
-  Make sure `npx cadl compile .` runs correctly.
-
-  ***Add cadl-csharp emitter***
-
-  Modify package.json, add one line under dependencies:
-
-```diff
-        "dependencies": {
-          "@cadl-lang/compiler": "^0.37.0",
-          "@cadl-lang/rest": "^0.19.0",
-          "@azure-tools/cadl-azure-core": "^0.9.0",
-+         "@azure-tools/cadl-csharp": "0.1.8"
-        },
-```
-
-        Run `npm install` again to install @azure-tools/cadl-csharp.
-
-  **Notes**: @azure-tools/cadl-csharp: "0.1.8" only works with @cadl-lang/compiler: "0.37.0" and @cadl-lang/rest: "0.19.0"
-  
-
-  ***Modify (or create) cadl-project.yaml, add one line under emitters:***
-
-  ```diff
-  emitters:
-+  "@azure-tools/cadl-csharp": true
-  ```
-
-  ***Configuration (optional)***
-
-  One can further configure the SDK generated, using the emitter options on @azure-tools/cadl-csharp.
-
-```
-emitters:
-  "@azure-tools/cadl-csharp":
-    namespace: "Azure.IoT.DeviceUpdate"
-    clear-output-folder: true
-```
+For first time to setup of a new sdk package, please verify you have met the prerequisites, including runtime environment, typespec project, sdk project folder. You can refer to [SDK Generation Prerequisites](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/DataPlaneCodeGeneration/AzureSDKGeneration_Prerequistites.md) to setup.
```

Comment:
Are these pre-reqs validated at runtime when we run the scripts? It is good to share these details in the beginning for people to read but if possible having these gates in place at runtime will also be useful.

Reply:
As @m-nash suggestion, we will sperate the requirement in another document, and just mention that customer to verify /setup those requirement before generate sdk. This QuickStart doc mainly focus on the SDK generation.

Reply:
@pallavit are you suggesting that if someone just tries to run `dotnet build /t:GenerateCode` they get a clear error message saying `Your nodejs version must be >= 16.x.x`?